pkg/utils/k8sutil.go

```diff
@@ -377,11 +376,9 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 	}
 
 	// update deployment for memory request
-	if funcObj.Spec.Memory != "" {
-		//mem value has already been validated before
-		quantity, _ := resource.ParseQuantity(funcObj.Spec.Memory)
+	if funcObj.Spec.Memory != (resource.Quantity{}) {
```

Comment:
oh of course, hah.  It just looks odd, I wonder why I don't see that more often ..